_dashboards/admin-ui-index/index-management.md

```diff
@@ -0,0 +1,223 @@
+nav_order: 10
+---
+
+# Index Management
+Introduced 2.5
+{: .label .label-purple }
+
+The Index Management section in the admin UI allows you to preform the operations available in the [Index API]({{site.url}}{{site.baseurl}}/api-reference/index-apis/index/) from OpenSearch Dashboards' web UI.
+
+## Index Policies
+
+[Policies]({{site.url}}{{site.baseurl}}/im-plugin/ism/policies/) are are configurations (JSON documents can also be uploaded) that define the following:
+
+1. The states that an index can be in, including the default state for new indexes. For example, you might name your states “hot,” “warm,” “delete,” and so on. For more information, see [States]({{site.url}}{{site.baseurl}}/im-plugin/ism/policies/#states).
+2. Any actions that you want the plugin to take when an index enters a state, such as performing a rollover. For more information, see [Actions]({{site.url}}{{site.baseurl}}/im-plugin/ism/policies/#actions).
+3. The conditions that must be met for an index to move into a new state, known as transitions. For example, if an index is more than eight weeks old, you might want to move it to the “delete” state. For more information, see [Transitions]({{site.url}}{{site.baseurl}}/im-plugin/ism/policies/#transitions).
+
+In other words, a policy defines the states that an index can be in, the actions to perform when in a state, and the conditions that must be met to transition between states.
+
+You have complete flexibility in the way you can design your policies. You can create any state, transition to any other state, and specify any number of actions in each state.
+
+To attach policies to indices:
+
+1. Select **Index policies** under **Index Management**.
+2. Choose the index or indexes that you want to attach your policy to.
+3. Choose Apply policy.
+4. From the Policy ID menu, choose the policy that you created.
+5. You can see a preview of your policy.
+6. If your policy includes a rollover operation, specify a rollover alias. Make sure that the alias that you enter already exists. For more information about the rollover operation, see rollover.
+7. Choose Apply.
+8. After you attach a policy to an index, ISM creates a job that runs every 5 minutes by default to perform policy actions, check conditions, and transition the index into different states. To change the default time interval for this job, see Settings.
+
+Policy jobs are not run if the cluster state is red.
+{: .note}
+
+## Managed Indices
+
+To attach policies to indicies:
+
+1. **Manage Indices** under **Index Management**.
+2. Choose the index or indices that you want to attach your policy to.
+3. Select the **Change policy** button.
+4. Choose Apply policy.
+
+## Indices
+
+Displays a list of indices in your OpenSearch cluster. Various properties are listed for each index including Heath (Green, Yellow, Red), if the index is managed by a policy, status, total size, size of primaries, total documents, deleted documents, primaries and replicas.
```

Comment:
First sentence is incomplete. What displays the list of indices? The **Index settings** window?